terraso_backend/apps/graphql/schema/visualization_config.py

```diff
@@ -101,17 +101,27 @@ class Meta:
 
     @classmethod
     def get_queryset(cls, queryset, info):
+        if info.field_name != "visualizationConfigs":
+            return queryset
+
```

Comment:
suggestion: add a comment explaining why this is only used for visualization configs.